tests/integration/workflows/helpers/mocks.py

```diff
@@ -48,6 +48,24 @@ def fake_download_file(workflow, name, url):
             )
         )
         return workflow.files[name]
+    if url == 'http://export.arxiv.org/e-print/1705.02541':
+        workflow.files[name] = pkg_resources.resource_stream(
+            __name__,
+            os.path.join(
+                '../fixtures',
+                '1407.7587v1'
```

Comment:
This should probably match with the URL that you're intercepting above. Also, better:
```python
workflow_files[name] = pkg_resources.resource_stream(
    __name__, os.path.join('..', 'fixtures', '1705.02541'))
```